tf_module/lambda-files/cloudwatch-papertrail.js

```diff
@@ -24,9 +24,29 @@ function getEnvVarOrFail(varName) {
     }
     return value;
 }
+// Should match for example: "[error] The database has exploded"
```

Comment:
This is a build artifact file, right? Should it be in the version control system at all?

Reply:
Yes, this is a constraint imposed on us by the following things:

1. We are relying on Github (instead of something like S3) for hosting the Terraform module files
2. We want to impose as few dependency requirements on the end users of the module as possible. (Ie. users of the module should not need to have Docker or typescript or npm, etc installed to build the compiled artifacts)

It seems like the least complicated choice at this time. If we had a reliable S3 bucket to store the Lambda source files we could get rid of having to store the compiled artifacts in source control.

Reply:
Right. This is fine for now. Would another option be to use the terraform module registry: https://registry.terraform.io/